demo/sections/components/DateRangeSelect.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <ComponentPage title="Date Range Select" :demos="[{ title: 'Date Range Select' }]">
+    <template #description>
+      <DemoState v-model:state="exampleState" v-model:disabled="disabled" />
```

Comment:
Not sure disabled work here either.  That would be a useful prop to have.

Reply:
You're right, another missing prop. Will fix